src/main/java/uk/gov/companieshouse/accounts/association/utils/MessageType.java

```diff
@@ -16,13 +16,13 @@ public enum MessageType {
 
     INVITE_MESSAGE_TYPE("associations_invite");
 
-    private final String messageType;
+    private final String messageTypeLocal;
```

Comment:
message type local does not suit this.
can we change it to something meaningful? like value.